<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExecuteWindow</class>
 <widget class="QWidget" name="ExecuteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioNormal">
           <property name="text">
            <string>separate files</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labelTraining">
             <property name="text">
              <string>Training set file: </string>
             </property>
             <property name="buddy">
              <cstring>comboBoxTraining</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxTraining"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="labelTesting">
             <property name="text">
              <string>Testing set file: </string>
             </property>
             <property name="buddy">
              <cstring>comboBoxTesting</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxTesting"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioCrossVal">
           <property name="text">
            <string>k fold cross-validation</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>file: </string>
             </property>
             <property name="buddy">
              <cstring>comboBoxCrossVal</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxCrossVal"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>k size: </string>
             </property>
             <property name="buddy">
              <cstring>kSize</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="kSize"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBoxCross">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelLearning">
         <property name="text">
          <string>Type of learning algorithm: </string>
         </property>
         <property name="buddy">
          <cstring>comboBoxAlgorithm</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxAlgorithm"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
